CONTRIBUTING.rst

```diff
@@ -0,0 +1,138 @@
+Contributing to towncrier
+=========================
+
+Want to contribute to this project? Great! We'd love to hear from you!
+
+As a developer and user, you probably have some questions about our
+project and how to contribute.
+In this article we try to answer these and give you some recommendations.
+
+
+Ways to contribute
+------------------
+
+There are several options to contribute to this project:
+
+* Open a new topic on our `GitHub discussion <gh_discussions>`_ page.
+
+  Tell us about your ideas or ask questions there.
+  Discuss with us the next Towncrier release.
+
+* Help or comment on our GitHub `issues`_ tracker.
+
+  There are certainly many issues where you can help with your expertise.
+  Or you would like to share your ideas with us.
+
+* Open a new `issue <issues>`_.
+
+  If you found a bug or have a new cool feature, describe your findings.
+  Try to be as descriptive as possible to help us understand your issue.
+
+* :ref:`Modify the code <modify_code>`.
+
+  If you would love to see the new feature in the next release, this is
+  probably the best way.
+
+
+.. _modify_code:
+
+Modifying the Code
+------------------
+
+The source code is managed using Git and is hosted on GitHub::
+
+    https://github.com/twisted/towncrier
+    git@github.com:twisted/towncrier.git
+
+
+We recommend the following workflow:
+
+#. Fork our project on GitHub using this link:
+   https://github.com/twisted/towncrier/fork
+
+#. Clone your forked Git repository (replace ``GITHUB_USER`` with your
+   account name on GitHub)::
+
+   $ git clone git@github.com:GITHUB_USER/python-semver.git
+
+
+#. In your local repository create a pull request:
+
+   a. Create a new branch with::
+
+      $ git checkout -b <BRANCH_NAME>
+
+   b. Write your test cases and run the complete test suite, see :ref:`testsuite`
+      for details.
+
+   c. Document any user facing changes in the ``README.rst`` file.
+
+   d. Create a newsfragment in ``src/towncrier/newsfragments/``.
+
+   e. Create a `pull request`_.
+      Describe in the pull request what you did and why. If you have open questions, ask.
+
+#. Wait for feedback. If you receive any comments, address these.
+
+#. After your pull request got accepted, delete your branch.
+
+
+.. _testsuite:
+
+Running the Test Suite
+----------------------
+
+We use the default unittest module and `tox`_ to run tests against all supported
```

Comment:
we use `twisted.trial` to run the test.

Reply:
Ahh right. My fault. Will correct that. :wink:

Reply:
Fixed in 601ea63.